libs/openFrameworks/gl/ofTexture.cpp

```diff
@@ -1128,35 +1141,50 @@ ofMesh ofTexture::getMeshForSubsection(float x, float y, float z, float w, float
 	// to constantly add a 2 pixel border on all uploaded images
 	// is insane..
 
-	GLfloat offsetw = 0.0f;
-	GLfloat offseth = 0.0f;
```

Comment:
hmm - I am really not super in favor of introducing this pattern across the whole core. 

If it is for optimization I wouldn't bother as the compiler will do the optimization for you.